SpiceSharp/Simulations/IntegrationMethods/States/StateManager.cs

```diff
@@ -0,0 +1,53 @@
+﻿using SpiceSharp.Algebra;
+
+namespace SpiceSharp.IntegrationMethods
+{
+    /// <summary>
+    /// Class responsible for managing state variables
+    /// </summary>
+    public class StateManager
+    {
+        /// <summary>
+        /// Number of states in the pool
+        /// </summary>
+        public int States { get; private set; }
```

Comment:
@svenboulanger  Please set initial value.

Reply:
Resharper suggests to not use it as the default is already `0`.